compiler/src/iree/compiler/Codegen/Dialect/LoweringConfig.td

```diff
@@ -11,62 +11,57 @@ include "iree/compiler/Codegen/Dialect/IREECodegenDialect.td"
 include "mlir/IR/EnumAttr.td"
 
 // List of pre-existing pipelines for translating executables.
-// TODO(#9047): Merge CPUDefault and CPUBuffersDefault pipelines. The latter one
-// is needed because of early bufferized ops. We're able to remove it if
-// upstream bufferization is used in this case.
 def CPU_Default
     : I32EnumAttrCase<"CPUDefault", 0>;
-def CPU_BufferOpsDefault
-    : I32EnumAttrCase<"CPUBufferOpsDefault", 1>;
 def CPU_DoubleTilingExpert
-    : I32EnumAttrCase<"CPUDoubleTilingExpert", 2>;
+    : I32EnumAttrCase<"CPUDoubleTilingExpert", 1>;
 def CPU_DoubleTilingPadExpert
-    : I32EnumAttrCase<"CPUDoubleTilingPadExpert", 3>;
+    : I32EnumAttrCase<"CPUDoubleTilingPadExpert", 2>;
 def CPU_ConvTileAndDecomposeExpert
-    : I32EnumAttrCase<"CPUConvTileAndDecomposeExpert", 4>;
+    : I32EnumAttrCase<"CPUConvTileAndDecomposeExpert", 3>;
 def CPU_TileFuseAndVectorize
-    : I32EnumAttrCase<"CPUTileFuseAndVectorize", 5>;
+    : I32EnumAttrCase<"CPUTileFuseAndVectorize", 4>;
 def CPU_BufferOpsTileAndVectorize
-    : I32EnumAttrCase<"CPUBufferOpsTileAndVectorize", 6>;
+    : I32EnumAttrCase<"CPUBufferOpsTileAndVectorize", 5>;
```

Comment:
I really don't like this kind of change... Do we have something similar to C++ enum in MLIR?

E.g.,

```
enum LoweringConfig : int32_t {
  CPUDefault = 0,
  CPUDoubleTilingExpert,
 ...
  SPIRVVectorizeWithWorkgroupMemory,
  None = 0xFF
};
```